### ISSUE TITLE
Create a fieldset component wrapper for nested fields (WCAG 2.1)

### ISSUE THREAD

**Issue by NomeQ** [interline-io/catenary#8]
Nested fields, such as groups of checkboxes and radio buttons, or multiple related fields, should be grouped together under a descriptive fieldset element: https://www.w3.org/WAI/tutorials/forms/grouping/

Create a wrapper/composable to allow these to be put together sensibly
